.github/workflows/ci-baseline.yml

```diff
@@ -32,36 +32,38 @@ jobs:
       fail-fast: false
       matrix:
         image:
-          - ubuntu-20.04-gnu_9-eigen_3.3.7-gtest
-          - ubuntu-20.04-clang_9-eigen_3.3.7-gtest
-          - fedora-34-gnu_11-eigen_3.3.7-gtest
-          - fedora-34-clang_12-eigen_3.3.7-gtest
-          #- intel_oneapi-eigen_3.3.7-gtest
+          - fedora-clang
+          - fedora-gnu
+          - ubuntu-clang
+          - ubuntu-gnu
+          #- intel-oneapi
         build_type:
           - Release
           - Debug
         cmake_extra_args:
           - ''
         include:
-          - image: ubuntu-20.04_gnu-11_eigen-3.3.7_gtest
+          - image: ubuntu-gnu
             build_type: Release
             cmake_extra_args: '-D CMAKE_CXX_STANDARD=20 -D PRESSIO_ENABLE_CXX20=ON'
-          - image: ubuntu-20.04_clang-12_eigen-3.3.7_gtest
+          - image: ubuntu-clang
             build_type: Release
             cmake_extra_args: '-D CMAKE_CXX_STANDARD=20 -D PRESSIO_ENABLE_CXX20=ON'
-          - image: fedora-34-gnu_11-eigen_3.3.7-gtest
+          - image: fedora-gnu
             build_type: Release
             cmake_extra_args: '-D CMAKE_CXX_STANDARD=20 -D PRESSIO_ENABLE_CXX20=ON'
-          - image: fedora-34-clang_12-eigen_3.3.7-gtest
+          - image: fedora-clang
             build_type: Release
             cmake_extra_args: '-D CMAKE_CXX_STANDARD=20 -D PRESSIO_ENABLE_CXX20=ON'
 
     runs-on: ubuntu-latest
-    container: pressiomodelreduction/${{ matrix.image }}
+    container:
+      image: ghcr.io/pressio/${{ matrix.image }}
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.github_token }}
```

Comment:
This is only necessary for private packages.

TODO: make pressio containers public and remove this.